code/modules/organs/external/_external.dm

```diff
@@ -156,23 +156,25 @@
 		return
 
 	var/burn_damage = 0
+	var/stun_time = 0
 	switch (severity)
 		if (EMP_ACT_HEAVY)
+			stun_time = 30 SECONDS
 			burn_damage = 30
 		if (EMP_ACT_LIGHT)
-			burn_damage = 15
-
-	var/mult = 1 + !!(BP_IS_ASSISTED(src)) // This macro returns (large) bitflags.
-	burn_damage *= mult/species.get_burn_mod(owner) //ignore burn mod for EMP damage
+			stun_time = 15 SECONDS
+			burn_damage = 30
```

Comment:
Should this not remain as 15 for EMP_ACT_LIGHT? burn_damage still affects implanted objects. Actually, because line 175 was removed, burn_damage does nothing now, so the var should either be removed, or these checks should be modified to keep the effect for implanted objects.